linting/requirements.txt

```diff
@@ -0,0 +1 @@
+sphinx-lint==0.6.7
```

Comment:
Doesn't runbot install the requirements specified in the root directory only?

Reply:
Also, `/linting/` -> `/tests/`.

Reply:
Nope, right now, they only install requirements if specified in the build step.
Since this sphinx-lint dependency is not required for all users of the doc, i thought of putting it in a dedicated requirements file